docs/releases/2.0.txt

```diff
@@ -401,6 +401,18 @@ For custom management commands that use options not created using
     class MyCommand(BaseCommand):
         stealth_options = ('option_name', ...)
 
+Indexes no longer accept optional arguments as positional arguments
```

Comment:
I guess "optional" could be removed; since all arguments are optional (unlike form fields where there are some required args), it's a bit redundant.